torchgeo/transforms/intensity.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""TorchGeo intensity transforms."""
+
+import torch
+from kornia.augmentation import IntensityAugmentationBase2D
+from torch import Tensor
+
+
+class ToDecibelScale(IntensityAugmentationBase2D):
```

Comment:
I would change the name to `PowerToDecibel`. Otherwise, when we add amplitude -> decibel, there will be a name conflict.